src/device_trezor/README.md

```diff
@@ -15,12 +15,22 @@ Please, refer to [monero readme](https://github.com/trezor/trezor-firmware/blob/
 
 ## Dependencies
 
-Trezor uses [Protobuf](https://protobuf.dev/) library. As Monero is compiled with C++14, the newest Protobuf library version cannot be compiled because it requires C++17 (through its dependency Abseil library).
-This can result in a compilation failure.
+Trezor uses [Protobuf](https://protobuf.dev/) library. Monero is now compiled with C++ 17 by default.
+Thus, protobuf v23 and higher has to be used. 
```

Comment:
Is v23 a necessity? We build release binaries with v21 (https://github.com/monero-project/monero/blob/master/contrib/depends/packages/native_protobuf.mk#L2) and it compiled with Trezor support enabled according to CI in this PR.

Reply:
No, it is not a hard requirement, it is changed now